pmd-core/src/main/java/net/sourceforge/pmd/renderers/internal/sarif/SarifLog.java

```diff
@@ -563,6 +568,8 @@ public int hashCode() {
             private String ruleset;
             @java.lang.SuppressWarnings("all")
             private Integer priority;
+            @java.lang.SuppressWarnings("all")
```

Comment:
Are those SuppressWarnings needed?

Reply:
I'm not actually sure but since the class was auto-generated throughout the whole class, just kept it consistent. I can go through the class and see what I could drop? It will create a large diff noise though so happy to take direction on it.